core/server/data/migrations/init/2-create-fixtures.js

```diff
@@ -10,6 +11,15 @@ module.exports = function insertFixtures(options) {
 
     return Promise.mapSeries(fixtures.models, function (model) {
         logging.info('Model: ' + model.name);
+
+        // The Post model fixtures need a `published_at` date, where at least the seconds
+        // are different, otherwise `prev_post` and `next_post` helpers won't workd with
+        // them.
+        if (model.name === 'Post') {
+            _.forEach(model.entries, function (post, key) {
```

Comment:
I think `key` should be renamed to `index` here to avoid confusion. General naming rules are that a `key` relates to the name of a value on an object (usually a string), e.g. `foo["bar"]` and `index` relates to the position in an array (always an integer), e.g. `foo[1]`.